catalog/transformer/catalog-transformer-csv-queryresponse/src/main/java/ddf/catalog/transformer/csv/CsvMetacardTransformer.java

```diff
@@ -43,12 +48,20 @@ public BinaryContent transform(Metacard metacard, Map<String, Serializable> argu
 
     Map<String, String> aliases =
         (Map<String, String>) arguments.getOrDefault("aliases", new HashMap<>());
-
+    String attributeString = (String) arguments.get("columnOrder");
```

Comment:
if no `columnOrder` arg is present, will this return an empty string or null?

Reply:
i think it actually returns null, so I should guard for that 👍